src/domain/backbone/entities/schemas/backbone.schema.ts

```diff
@@ -1,24 +1,11 @@
-import { Backbone } from '@/domain/backbone/entities/backbone.entity';
-import { JSONSchemaType } from 'ajv';
+import { z } from 'zod';
 
-export const BACKBONE_SCHEMA_ID =
-  'https://safe-client.safe.global/schemas/backbone/backbone.json';
-
-export const backboneSchema: JSONSchemaType<Backbone> = {
-  $id: BACKBONE_SCHEMA_ID,
-  type: 'object',
-  properties: {
-    name: { type: 'string' },
-    version: { type: 'string' },
-    api_version: { type: 'string' },
-    secure: { type: 'boolean' },
-    host: { type: 'string' },
-    headers: { type: 'array', items: { type: 'string' } },
-    settings: {
-      type: 'object',
-      propertyNames: { type: 'string' },
-      required: [],
-    },
-  },
-  required: ['name', 'version', 'api_version', 'secure', 'host'],
-};
+export const BackboneSchema = z.object({
+  name: z.string(),
+  version: z.string(),
+  api_version: z.string(),
+  secure: z.boolean(),
+  host: z.string(),
+  headers: z.array(z.string()),
+  settings: z.record(z.unknown()),
```

Comment:
I think `headers` and `settings` are optional, as they are not included in the the JSON Schema `required` array.

Reply:
Good catch! I missed the `required` array. I've made them both `optional()` in 7b950b6552c8ad5faee24f60e042fe20c2bad7ef.